WEB-INTEGRATION.md

```diff
@@ -390,21 +392,25 @@ registration context; that is, the context in which the class is constructed.
 - [`ReportingObserver`](https://w3c.github.io/reporting/#reportingobserver)
   [\[REPORTING\]](https://w3c.github.io/reporting/)
 
-> TODO: Due to concerns about observers leading to memory leaks, an alternative
-> option is to not use the registration context, and instead call the observer's
-> callback with the empty context. This is still under discussion. 
+> [!IMPORTANT]
+> Due to concerns about observers leading to memory leaks, an alternative
+> option is to not use the context from when the observer was created, and instead call the observer's
+> callback with the empty context. This is coherent with the general design direction
+> of using the dispatch context, except that we would say that these observers are always
+> triggered as consequences of browser-internal code.
```

Comment:
We should probably update the document with this behavior. The only thing remaining to discuss is whether observers would have a fallback context, right?

Reply:
Ok, I'll keep some version of this paragraph as the motivation though.